packages/rocketchat-api/server/v1/groups.js

```diff
@@ -435,16 +435,7 @@ RocketChat.API.v1.addRoute('groups.setType', { authRequired: true }, {
 
 RocketChat.API.v1.addRoute('groups.unarchive', { authRequired: true }, {
 	post: function() {
-<<<<<<< HEAD
-		const findResult = findPrivateGroupByIdOrName({ roomId: this.bodyParams.roomId, userId: this.userId });
-
-		//The find method returns either with the group or the failure
-		if (findResult.statusCode) {
-			return findResult;
-		}
-=======
 		const findResult = findPrivateGroupById({ roomId: this.bodyParams.roomId, userId: this.userId, checkedArchived: false });
```

Comment:
You missed the change here I believe, shouldn't it be using `findPrivateGroupByIdOrName` now?